src/main/webapp/WEB-INF/pages/macros/resourcesTable.ftl

```diff
@@ -81,15 +81,13 @@ resourcesTable macro: Generates a data table that has searching, pagination, and
                     }
                 },
                 "aoColumns": [
-                    { "sTitle": "<@s.text name="portal.home.logo"/>", "bSearchable": false, "bVisible": <#if shownPublicly>true<#else>false</#if> },
```

Comment:
In home section: column logo still exists and column names were shifted 
In Resource Management: column names are also wrong, names to do not match the column info.

Reply:
It is true. This error was fixed.